test/gtest/uct/uct_test.cc

```diff
@@ -156,6 +156,12 @@ uct_test::entity* uct_test::create_entity(size_t rx_headroom) {
     return new_ent;
 }
 
+uct_test::entity* uct_test::create_entity(uct_iface_params_t &params) {
```

Comment:
should be const uct_iface_params_t& params
also, the old create_entity(rx_headroom) should be changed to call this function